jobs/models.py

```diff
@@ -1,3 +1,62 @@
+from django.utils import timezone
+
 from django.db import models
+from django_countries.fields import CountryField
+from djangogirls import settings
+
+
+class Company(models.Model):
+
+    class Meta:
+        verbose_name = "Company"
+        verbose_name_plural = "Companies"
+        ordering = ['name']
+
+    name = models.CharField(max_length=500, unique=True)
+    website = models.URLField()
+
+    def __unicode__(self):
+        return self.name
+
+class Job(models.Model):
+
+    class Meta:
+        unique_together = (("company", "title"),)
+        ordering = ['-post_date']
+
+    title = models.CharField(max_length=500)
+    company = models.ForeignKey('Company', related_name="company")
+    contact_email = models.EmailField(max_length=254)
+    city = models.CharField(max_length=100)
+    country = CountryField()
+    description = models.TextField(max_length=5000)
+    reviewer = models.ForeignKey(
+        settings.AUTH_USER_MODEL, 
+        related_name="reviewer",
```

Comment:
related_name='jobs'
related_name is the name that you'd use in the backwards relationship - so if you've assigned user1 to job1 and job2, you would do user1.jobs to get a list of all the jobs that user is associated with.  Currently you'd do user1.reviewer which would give you back a list of jobs, which isn't quite what you'd expect if you were inspecting the user model and saw a propery called reviewer.